plugins/coverbrowser.koplugin/main.lua

```diff
@@ -139,7 +139,7 @@ function CoverBrowser:addToMainMenu(menu_items)
                     G_reader_settings:delSetting("home_dir_display_name")
                     if FileManager.instance then FileManager.instance:reinit() end
                 else
-                    G_reader_settings:saveSetting("home_dir_display_name", "Home")
+                    G_reader_settings:saveSetting("home_dir_display_name", _("Home"))
```

Comment:
I don't think we should save to settings a translatable string (if I switch language from English to French, I expect to see "Accueil" and not the value "Home" it has when i was using English).

I think this change will need a little more work than what you intended to do :|
If we are going to use the main translatable string for "Home", we should probably just save the boolean `true` here - and allow for setting it to a string like `"~"` for users who want something else.
Which means some more work on the code that abbreviate the path to use that string, or the translatable `_("Home")`, depending on the setting type.

(Or simplify all that but not using any prefix, but needs comments on my other comment :)

Reply:
I’ll look into this more, I think not all users would want Home, I’ll see if I can figure out how to let users input a string for it instead, then it could be a simple “Shorten home directory”, click that and it brings up a text field or something of the sorts

Reply:
Don't rush into adding a InputDialog. It might be a little overkill for such a little setting.
People could and still can override it by editing manually the setting into their settings.reader.lua.
I realize that currently, we too are overriding the user specific setting when toggling that menu item, with a `~`. So, you were just changeing the overriding value from `~` to `Home`.

What do you think about my suggestion of going with nothing, from a UI/third audience point of view?
You would display `_("Home")` only when in home_dir, and a relative path (without leading /) when in a sub directory (without a `Home/` prefix).
(and when this checkbox is unchecked, we would display always the full path).

I would be fine with that, and it would make things simpler - just a boolean setting (while still allowing people to use a string to get something else because it's currently possible).

Reply:
Actually yes, I like that idea, _("Home") for the Home and / for others, I'm keen, don't have time to write that right now as it's getting late here, unless you have time? :D

Reply:
I can have a look. @Frenzie : any opinion on going with that (also explained in https://github.com/koreader/koreader/pull/5508#issuecomment-544232071).

Reply:
On Kobo that could make a lot of sense, less sure about other platforms.

One thing to keep in mind is that not displaying a home prefix would be antithetical to breadcrumbs, also see <https://github.com/koreader/koreader/issues/5037#issuecomment-495230466>.

Reply:
So, that kept in mind :), what do we want?
I guess once a home_dir is set, users rarely goes outside (and when they will, they'll see the full path with a leading /).
But when inside my home_dir, do I need to always see the home_dir breadcrumb/anchor (whether it's `~/` or `Home/` or `🏠/`) ? I don't think so. It's noise. `Home/` is too long for me, `~/` is short and so better, but _nothing_ feels even better.
So, when navigating home_dir/authors/John Martin/Novels/, we'll see
`authors/John Martin/Novels`
or do we really need to see:
`Home/authors/John Martin/Novels` or
`~/authors/John Martin/Novels`  or
`🏠/authors/John Martin/Novels` ?

> less sure about other platforms

Why would that be different? A set home_dir is a home_dir no matter the platform :) If there's a secondary location where books are kept, well, that's outside the home_dir, so we'll display a full path. If a user is confused and have to work with book in multiple locations, the only option, like now, is to not check that option of shortening.

Reply:
> or do we really need to see:

Need not necessarily, but I think it's at best inconvenient not to. If you have an interaction pattern like breadcrumbs that normally works, breaking it to get rid of some noise for a nice clean look isn't a good thing.

Dolphin actually does something very much like what you're proposing and it's really annoying; its biggest flaw perhaps. Any direct subfolder of home doesn't display home in the breadcrumbs, completely breaking the normal `>` quick folder switching functionality. Windows Explorer/Directory Opus does it best.

But I suppose we can table my argument for the moment because there's no such thing as breadcrumbs.

I do like to know where I am though (i.e., Home, not nowhere).

> Why would that be different?

Because Kobo's /mnt/onboard path is very ugly. ;-)

Reply:
> I do like to know where I am though

So, that's where the new translatable string `_("Home")` would be used, and only when just in home_dir (I was fine without any, but ok).
(That will be confusing if a user create a subdirectory named "Home" thus :)